src/passthrough/async_io.rs

```diff
@@ -97,8 +97,9 @@ impl<D: AsyncDrive> PassthroughFs<D> {
         ctx: &Context,
         inode: Inode,
         flags: u32,
-        fuse_flags: u32,
+        _fuse_flags: u32,
     ) -> io::Result<(Option<Handle>, OpenOptions)> {
+        // FIXME: handle FOPEN_IN_KILL_SUIDGID properly
```

Comment:
Do we need to enhance the io_uring subsystem to support this feature?

Reply:
yup, async IO, we are not sure if it is still the same thread when reverting the thread cap changes in IO finish callback.